src/main/java/org/moltimate/moltimatebackend/repository/QueryResponsetDataRepository.java

```diff
@@ -0,0 +1,8 @@
+package org.moltimate.moltimatebackend.repository;
+
+import org.moltimate.moltimatebackend.dto.Alignment.QueryResponseData;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface QueryResponsetDataRepository extends JpaRepository<QueryResponseData, Long> {
```

Comment:
It looks like `QueryResponsetDataRepository` has a type

Reply:
Done